amgut/lib/data_access/env_management.py

```diff
@@ -137,6 +138,29 @@ def make_settings_table():
         TRN.add(sql, args)
 
 
+def add_labadmin_superuser():
+    """Adds a new labadmin user names 'master' to the database and grants
+       highest privileges to this user. Password is 'password'."""
+    os.environ['PGPASSWORD'] = AMGUT_CONFIG.password
+    command = ['psql',
+               '--dbname', AMGUT_CONFIG.database,
+               '--host', AMGUT_CONFIG.host,
+               '--port', str(AMGUT_CONFIG.port),
+               '--username', AMGUT_CONFIG.user,
+               '-c', ("INSERT INTO ag.labadmin_users (email, password) "
+                      "VALUES ('master', '$2a$10$2.6Y9HmBqUFmSvKCjWmBte70WF."
+                      "zd3h4VqbhLMQK1xP67Aj3rei86'); "
+                      "INSERT INTO ag.labadmin_users_access (access_id, email)"
+                      " VALUES (7, 'master');")]
+    proc = Popen(command, stdin=PIPE, stdout=PIPE)
+    retcode = proc.wait()
+    os.environ['PGPASSWORD'] = ''
+    if retcode != 0:
+        raise RuntimeError(("Could not add labadmin superuser 'master' to "
+                            "database %s: retcode %d") %
+                           (AMGUT_CONFIG.database, retcode))
+
+
 def populate_test_db():
```

Comment:
Add the creation of the master user inside this function, since it is only relevant for the test_db.